care/emr/utils/compute_observation_interpretation.py

```
@@ -18,6 +18,9 @@ def compute_observation_interpretation(model_instance, metrics_cache):
         if interpretation:
             model_instance.interpretation = interpretation
             model_instance.reference_range = ranges
+        else:
+            model_instance.interpretation = "Normal"
```

defaulting to `Normal` might be misleading.